source/particle/property/integrated_strain.cc

```diff
@@ -0,0 +1,101 @@
+/*
+  Copyright (C) 2015 - 2016 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file doc/COPYING.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#include <aspect/particle/property/integrated_strain.h>
+
+namespace aspect
+{
+  namespace Particle
+  {
+    namespace Property
+    {
+      template <int dim>
+      void
+      IntegratedStrain<dim>::initialize_one_particle_property(const Point<dim> &,
+                                                              const Vector<double> &,
+                                                              const std::vector<Tensor<1,dim> > &,
+                                                              std::vector<double> &data) const
+      {
+        for (unsigned int i = 0; i < Tensor<2,dim>::n_independent_components ; ++i)
```

Comment:
This is analogous to the finite strain cookbook for compositional fields. I do not quite recall the reason (this central part is 6 months old now), but I think we had trouble multiplying a symmetric tensor from both sides with anti-symmetric tensors and store the result in a symmetric tensor, although the result should be symmetric again (@jdannberg, do you remember exactly why we decided to go for full tensors?)